sdk/state/state.go

```diff
@@ -133,6 +143,20 @@ func (c *Channel) responderSigner() *keypair.FromAddress {
 	}
 }
 
+func (c *Channel) initiatorClaimAmount() int64 {
```

Comment:
We should name these functions `initiatorBalanceAmount` using the term Balance instead of Claim, since we don't use the term claim anywhere else.

Reply:
https://github.com/stellar/experimental-payment-channels/pull/95/commits/02f9d38f37cd5e88b3d0c739b0bb9debb6753faf